.postcssrc.json

```diff
@@ -6,7 +6,7 @@
       "mediaQuery": true
     },
     "autoprefixer": {
-      "browsers": ["last 2 versions"],
+      "overrideBrowserslist": ["last 2 versions"],
```

Comment:
was looking at what this do in the [docs](https://github.com/postcss/autoprefixer#options), looks like its use is discouraged? Looks like the best practice they recommend is to use .browserslistrc config or browserslist key in package.json

Reply:
I tried it out and it looks like that might affect the js file contents? The docs say that it's recommended since it also can apply to babel and stylelint. But since we weren't doing it before, it's possible this could change some behavior? I took a sha1 hash of answers.js before and after adding browserslist to the package.json and got different values, while the hash for answers.css was identical before and after. I want to push off adding a shared browserslist setting to later if that's okay?

Reply:
sure, maybe make an item or add to the idea sheet?

Reply:
added!